Makefile.PL

```diff
@@ -41,6 +41,7 @@ test_requires 'Test::NoWarnings';
 recommends 'DBD::mysql';
 recommends 'DBD::Pg';
 recommends 'DBD::SQLite';
+recommends 'Net::Statsd';
```

Comment:
Today Zonemaster-Backend can be installed on FreeBSD by packages only (except Zonemaster itself). That makes it possible to package it. We should not add dependencies to CPAN.

Reply:
>  That makes it possible to package it

I am assuming you mean not possible.  What change are you requesting here, removing the dependency from the `recommends` or totally removing it? Why is it an issue to have dependencies to CPAN (given that we are already using that to install Zonemaster)?

Reply:
Are you saying that you want Zonemaster being package in the ports and that make it not possible?

Reply:
Yes. To have Zonemaster in ports (available as package) we need to have all dependencies in ports (as package). As it is now all dependencies can be installed by FreeBSD `pkg install`.

Reply:
Can that issue be addressed when we will be packaging Zonemaster for Freebsd? (e.g. by creating the missing port, I had to do a similar thing last week when working on [Fedora packaging](https://gitlab.rd.nic.fr/zonemaster/packages/fedora) or just disabling the feature for Freebsd) If not what is your suggestion here? (I don't see any of the [statsd perl clients](https://metacpan.org/search?q=statsd) listed in the ports https://www.freebsd.org/cgi/ports.cgi?query=Statsd&stype=all&sektion=all)

Reply:
It is not realistic that we should create a port/package of `Net::Statsd` for the FreeBSD ports system.

If it is removed from "recommends" in `Makefile.PL` it should work. Then it is made true optional. You have to install it if you want to use it.

We have already decided that we should avoid adding dependencies that are not available as a binary package, and that is independent of OS. Can you see any alternative?

Reply:
> Can you see any alternative?

I tried to find other metrics modules (Measure::Everything, Influx, Net::Prometheus, Net::Statsd) in the freebsd port and did not find any, so I don't think so.

> If it is removed from "recommends" in Makefile.PL it should work. 

Then I'll do that.

> we should avoid adding dependencies that are not available as a binary package

For the record this package is available as binary package for debian / ubuntu and fedora.

Reply:
> > we should avoid adding dependencies that are not available as a binary package
> 
> For the record this module is available as binary package for debian / ubuntu and fedora.

Then hopefully it moves into FreeBSD ports too.